src/execution/window_segment_tree.cpp

```diff
@@ -500,6 +495,37 @@ void WindowSegmentTree::Evaluate(const idx_t *begins, const idx_t *ends, Vector
 	}
 	FlushStates(true);
 
+	//	Seggond pass: aggregate the ragged leaves
```

Comment:
Second? :)

Reply:
It is now.  My push I forgot to fix the comments.